.github/workflows/helm-chart-release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Helm Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.1.0
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
@yonatankahana - could you raise an issue with the gh-admin to setup GITHUB_TOKEN to this repo. Probably something that comes from the organization's secrets. (similar to the PR that you raised to resolve the branch protection rules)

Reply:
Or, are we good with the default permissions availble on GITHUB_TOKEN enabled for github actions. https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Reply:
hope that i understand you correctly, do you mean that the token will be k8s bot token?
if so, this may be related? https://github.com/kubernetes/test-infra/pull/20291, https://github.com/kubernetes/test-infra/pull/20425

Reply:
yes. on the lines of the above. Let me go ahead with this merge and see if we are good based on the discussions in the above PRs.

Reply:
for example, they use the default token here: https://github.com/kubernetes/kube-state-metrics/releases